geoportailv3/static/js/draw/featurepopupdirective.js

```diff
@@ -95,6 +126,19 @@ app.FeaturePopupController.prototype.close = function() {
 
 
 /**
+ * @return {boolean} return true if is editable by the user
+ * @export
+ */
+app.FeaturePopupController.prototype.isEditable = function() {
+  if (goog.isDefAndNotNull(this.feature) &&
+      this.feature.get('__source__') == 'mymaps') {
+    return this.appMymaps_.isEditable();
+  }
+  return true;
```

Comment:
I think that if no feature exists, it's not important what is returned. Don't you think ?